yw_website/yw_db/migrations/0001_initial.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 2.1.1 on 2018-11-11 23:42
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='File',
+            fields=[
+                ('checksum', models.CharField(max_length=20, primary_key=True, serialize=False)),
+                ('input_data', models.FileField(blank=True, upload_to='recon_files/')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Run',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('time_stamp', models.DateTimeField()),
+                ('yw_recon_output', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='RunFile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('uri', models.CharField(max_length=255)),
+                ('file_name', models.CharField(max_length=255)),
+                ('time_stamp', models.DateField()),
+                ('file_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='yw_db.File')),
+                ('run_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='yw_db.Run')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Tag',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('tag_type', models.CharField(choices=[('w', 'Workflow'), ('v', 'Version'), ('r', 'Run')], default='w', max_length=1)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Version',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('script_check_sum', models.CharField(max_length=20)),
+                ('yw_model_check_sum', models.CharField(max_length=20)),
+                ('yw_model_output', models.CharField(max_length=255)),
+                ('yw_graph_output', models.TextField()),
+                ('last_modified', models.DateField()),
+                ('version_tag', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='yw_db.Tag')),
```

Comment:
there is a many to many relationship with tags, need to remove `version_tag` and use an intermediary table